packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Updated
+  * Changed orion-fusion-theme status view bottom and top padding to reduce white space.
```

Comment:
```suggestion
```

why do we need change log update on core-docs ..??

Reply:
removed